src/h5web/test-utils.tsx

```diff
@@ -18,8 +19,11 @@ export function renderApp(): RenderResult {
 }
 
 export async function selectExplorerNode(path: string): Promise<void> {
-  for await (const name of path.split('/')) {
-    fireEvent.click(await screen.findByRole('treeitem', { name }));
+  const [name, ...remainingSegments] = path.split('/');
+  fireEvent.click(await screen.findByRole('treeitem', { name }));
+
+  if (remainingSegments.length > 0) {
+    await selectExplorerNode(remainingSegments.join('/'));
```

Comment:
Using recursion instead of a loop. Feels nicer.

For some reason, replacing this `fireEvent` with `userEvent` introduces a regression with one specific `NexusPack` test. React logs a warning "An update to MappedHeatmapVis inside a test was not wrapped in act(...).". I get what it means (the heatmap config is updated in a `useEffect` after the test has completed), but I don't get why I don't see the same regression in other tests, and why this particular change causes the regression. So... after an hour of investigation, it's time to give up.